tsconfig.json

```diff
@@ -27,15 +27,16 @@
       "interactions/*": ["extensions/interactions/*"],
       "filters/*": ["core/templates/filters/*"],
       "static/*": ["third_party/static/*"],
-      "tests/*": ["core/templates/tests/*"]
+      "tests/*": ["core/templates/tests/*"],
+      "test_files/*": ["scripts/linters/test_files/*"]
```

Comment:
@ankita240796 I've added this file path here becuase we need to compile test files too for testing.